src/main/java/de/tum/cit/aet/artemis/communication/web/FaqResource.java

```diff
@@ -72,18 +75,16 @@ public FaqResource(CourseRepository courseRepository, AuthorizationCheckService
      * @throws URISyntaxException if the Location URI syntax is incorrect
      */
     @PostMapping("courses/{courseId}/faqs")
-    @EnforceAtLeastInstructor
+    @EnforceAtLeastTutorInCourse
```

Comment:
Could you please have another look at the permissions in this Resource? 
This PR description says that editors+ should be allowed to propose FAQs but here it seems like tutors+ are allowed to even `create` FAQs? 

Do I misunderstand something here?

Reply:
I do now see that `checkPriviledgeForAcceptedElseThrow` seems to cover the permission check. Still the inconsistency with tutor vs editor exists

Is there a particular reason why this is handled in one endpoint? I think it would be cleaner to have a separate endpoint for proposing a faq 😅

Reply:
Ah well, just the motivation and description seems to differ from each other, so I guess the tutor change is intended 😅 
So it would be good if the description could be updated

Reply:
I used the same endpoints for create and update, cause otherwise i simply duplicate the code but only change the access check with @EnforceAtleastInstructor / @EnforeAtleastEditor and the name of the endpoint create/propose or update/updateProposed